typing_extensions/src/typing_extensions.py

```diff
@@ -2096,6 +2097,100 @@ def __eq__(self, other):
 
     TypeGuard = _TypeGuard(_root=True)
 
+
+if hasattr(typing, "LiteralString"):
+    LiteralString = typing.LiteralString
+elif sys.version_info[:2] >= (3, 7):
+    # Vendored from cpython typing._SpecialFrom
+    class _SpecialForm(typing._Final, _root=True):
```

Comment:
Note that it also copies `_SpecialForm` class.

Reply:
Ah right, I knew I had copied it in some other place :)

Reply:
Oh wait, this is going to merge conflict in a bad way with the Never change. Maybe I should land that one first, then use the factored out _SpecialForm from that PR.